includes/checkout.php

```diff
@@ -291,9 +285,90 @@ function pmprogl_pmpro_after_checkout($user_id, $morder)
 	wp_cache_delete( $cache_key . '_all', 'pmpro' );
 	wp_cache_delete( $cache_key . '_active', 'pmpro' );
 
-  // update user data and call action
+	// Update user data and call action.
 	pmpro_set_current_user();
 
+	// Don't send the default checkout emails to user or admin.
+	add_filter( 'pmpro_send_checkout_emails', '__return_false', 15 );
+
+	// Send the checkout emails to gift purchasers, admin, and gift recipient (optional).
+	global $current_user;
+
+	// Get the gift code from the database.
+	$code = $wpdb->get_var("SELECT code FROM $wpdb->pmpro_discount_codes WHERE id = '" . intval( $code_id ) . "' LIMIT 1");
+
```

Comment:
@dparker1005 Are you talking about having a `$code` check here to make sure it was found?

RE: https://github.com/strangerstudios/pmpro-gift-levels/pull/57/files#r704617392

Reply:
Nevermind me on this one about the $code check